app/policies/resource_policy.rb

```diff
@@ -0,0 +1,12 @@
+class ResourcePolicy < ApplicationPolicy
```

Comment:
Yes, totally, just wanted to start with the bare minimum since I don't have any design or anything for the other actions.